NOTICE

```diff
@@ -1,2 +1,14 @@
 Docker Client
-Copyright (c) 2014 Spotify AB
+This software was originally developed by Spotify. The portions originating
+from Spotify is copyright Spotify AB.
+
+Substantial additions to this project has been made by the following
+contributions, who retain copyright in their contributions. For details
```

Comment:
"contributions, who retain copyright in their contributions": change the first "contributions" to "contributors"?